1.cellprofiler_ic/scripts/nf1_ic.py

```diff
@@ -20,27 +20,35 @@
 # In[2]:
 
 
-path_to_pipeline = pathlib.Path("NF1_illum.cppipe").absolute()
-
 plates_info_dictionary = {
     "Plate_1": {
-        "path_to_images": pathlib.Path("../0.download_data/Plate_1/").absolute(),
-        "path_to_output": pathlib.Path("Corrected_Plate_1").absolute(),
+        "path_to_pipeline": pathlib.Path("NF1_illum_Plates_1_2.cppipe").absolute().resolve(),
+        "path_to_images": pathlib.Path("../0.download_data/Plate_1/").absolute().resolve(),
+        "path_to_output": pathlib.Path("Corrected_Plate_1").absolute().resolve(),
     },
     "Plate_2": {
-        "path_to_images": pathlib.Path("../0.download_data/Plate_2/").absolute(),
-        "path_to_output": pathlib.Path("Corrected_Plate_2").absolute(),
+        "path_to_pipeline": pathlib.Path("NF1_illum_Plates_1_2.cppipe").absolute().resolve(),
+        "path_to_images": pathlib.Path("../0.download_data/Plate_2/").absolute().resolve(),
+        "path_to_output": pathlib.Path("Corrected_Plate_2").absolute().resolve(),
+    },
+    "Plate_3": {
+        "path_to_pipeline": pathlib.Path("NF1_illum_Plate3.cppipe").absolute().resolve(),
+        "path_to_images": pathlib.Path("../0.download_data/Plate_3/").absolute().resolve(),
+        "path_to_output": pathlib.Path("Corrected_Plate_3").absolute().resolve(),
```

Comment:
I don't think that `.absolute()` is necessary since `.resolve()` already returns the full path. Here's the [documentation](https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve)

also to add a bit of security, you can use the `strict=True` parameter as it checks if the file exists before generating an absolute path. if the file happens to not exist, then a `FileNotFoundError` will be raised through `resolve()`

Reply:
Ahh okay I wasn't sure if I needed to have resolve after or by itself. I will correct this and add the `strict=True`!

Reply:
See [bba1f03](https://github.com/WayScience/nf1_cellpainting_data/pull/7/commits/bba1f03707a1e6adb6fa16bbedbd7137a606272c) for change.